Makefile

```diff
@@ -1,18 +1,35 @@
+CACHITO_COMPOSE_ENGINE ?= docker-compose
+ifeq ($(CACHITO_COMPOSE_ENGINE), docker-compose)
+	DOWN_OPTS=-v
```

Comment:
If we're not using named volumes, you probably don't need `-v`, right? But it doesn't hurt.